posts/taking-chatgpt-for-an-impromptuspin.md

```diff
@@ -0,0 +1,102 @@
+
+When asked, ChatGPT's narrative is starkly different:
+
+```console
+Tell me the origins of ChatGPT
+
+---
+
+ChatGPT, also known as GPT-3, is a language generation model developed by OpenAI. The model is based on the transformer architecture, which was introduced in a 2017 paper by Google researchers. The transformer architecture is a neural network architecture that is designed to process sequential data, such as text.
+
+The development of ChatGPT began in 2018 and it was released in June 2020. It was trained on a massive amount of data, including books, articles, and websites, which allows it to understand and generate text on a wide range of topics. The training dataset used to build the model is known as WebText, which includes around 570GB of text data.
+
+ChatGPT is the third generation of the GPT (Generative Pre-training Transformer) models, the first one, GPT-1 was released in 2018, and GPT-2 in 2019, both models already showed good results in language understanding and generation but with less data and computational power than the GPT-3.
+
+ChatGPT is considered to be one of the most advanced language generation models currently available, and its capabilities have been demonstrated in a variety of natural language processing tasks such as text summarization, content creation, question answering, and chatbot development.
+```
+
+We know ChatGPT generated the tale of its upbringing. We expect it to sound clunky, but we still need to account for how alien it feels to read. Imagine asking someone, "So tell me about yourself; where are you from?" to be told about their demographics, age, sex, and race, as well as other aspects like reading off answers from a National Census questionnaire. Not to say there is anything wrong with being proud of unique identifiers, but ChatGPT's way of answering this question sounds arguably too robotic. But, perhaps, that is a litmus test for dealing with an AI.
+
+## Conclusion
+
+As [Microsoft continues to funnel money into the ChatGPT project](https://edition.cnn.com/2023/01/23/tech/microsoft-invests-chatgpt-openai/index.html) and [Google has its version of "Deepmind Chat" coming soon](https://www.techradar.com/news/googles-deepmind-promises-chatgpt-rival-soon-and-it-could-be-better-in-one-key-way), it is clear the days of artificial intelligence are only just beginning. Many industries have yet to see automation like this before and will face a [partial or complete overhaul](https://80000hours.org/2015/02/which-careers-will-be-automated/) soon.
+
+```console
+Hello Brave New World?
```

Comment:
I love this response! It really ties in with how alien, and foreign ChatGPT's self-generated origin story is!